modules/06_Gateway_Policies/06_1_Gateway_Policies_Lab.adoc

```diff
@@ -4,6 +4,8 @@
 :linkattrs:
 :data-uri:
 
+
```

Comment:
It looks like your changes made it into another PR, so please just close this one